CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+## Contributing
+
+Like any other open source projects, there are multiple ways to contribute to this project:
+
+* As a developer, depending on your skills and experience,
+* As a user who enjoys the project and wants to help.
+
+##### Reporting Bugs
+
+If you found something broken or not working properly, feel free to create an issue in Github with as much information as possible, such as logs and how to reproduce the problem. Before opening the issue, make sure that:
+
+* You have read this documentation,
+* You are using the latest version of project,
+* You already searched other issues to see if your problem or request was already reported.
+
+##### Improving the Documentation
+
+You can improve this documentation by forking its repository, updating the content and sending a pull request.
+
+
+#### We ❤️ Pull Requests
+
+A pull request does not need to be a fix for a bug or implementing something new. Software can always be improved, legacy code removed and tests are always welcome!
+
+Please do not be afraid of contributing code, make sure it follows these rules:
+
+* Your code compiles, does not break any of the existing code in the master branch and does not cause conflicts,
+* The code is readable and has comments, that aren’t superfluous or unnecessary,
+* An overview or context is provided as body of the Pull Request. It does not need to be too extensive.
+
+Extra points if your code comes with tests!
+
+For more information please check [README](README.md)
```

Comment:
added link to readme

Reply:
I mean, add instead a link **in** the `README.md` file, -> **to** `CONTRIBUTING.md`.
`README.md` is displayed in the github repository, it's like the starting point for all documentation.

Reply:
@ahmetcetin, could you, please, remove `For more information please check [README](README.md)` from `CONTRIBUTING.md` and create new section in the `README.md` file named `Contributing`, where you would create link to `CONTRIBUTING.md`?

Thanks! :smiley:

Reply:
updated readme and contributing as per your request.
Cheers,
Ahmet